### ISSUE TITLE
[Help] Chrome OS/Chromebook how to port forward the extension correctly.

### ISSUE THREAD

**Issue by Blue-EyesWhiteDragon** [kzahel/web-server-chrome#80]
I have no idea if this is already been said but oh well.

If you're on **Chrome OS/Chromebook** you can't use **port 80** with "200 OK!" (unless in dev mode I think) however you can do the following (this example is with the **BT Home Hub 5**, you'll have to work out where your **port forward** settings and what the alternative methods to what I use in my examples are for your router model):

Now assuming you have set up your Chrome OS device/Chromebook with a static ip (if you don't know how to do that then a google search will help you there).

#### You can create the following "200 OK" option by clicking on the button the arrow points to.

![1](https://cloud.githubusercontent.com/assets/7526156/20025399/e2a48528-a2e5-11e6-910d-b5f32e107425.png)

#### Inside that menu there's an option that looks similar (depending on router model) to the following: 
![2](https://cloud.githubusercontent.com/assets/7526156/20025422/16115684-a2e6-11e6-9fcb-831fe3f0d055.png)

#### Before continuing, make sure that your 200 OK is set up to like so:

![3](https://cloud.githubusercontent.com/assets/7526156/20025423/1a4d9faa-a2e6-11e6-8968-5f672b8db442.png)

#### After that do the following (depending on what your router model has in the port forward settings) and click apply.

![4](https://cloud.githubusercontent.com/assets/7526156/20025424/1c3c311e-a2e6-11e6-8dcc-80a1fa75289e.png)

#### Select "200 OK" or whatever you called the setting you configured and if you've named the device on your router you can select that from the dropdown menu or alternatively type in the device's ip address.

![1 - edited](https://cloud.githubusercontent.com/assets/7526156/20025726/5042d960-a2e9-11e6-9a94-8e411b439d54.png)

#### Type in your external ip address (you can google "my ip address" and it'll give you it) and boom!

![5](https://cloud.githubusercontent.com/assets/7526156/20025428/22e3108c-a2e6-11e6-8dd4-bc24c4a56d73.png)

#### The following screenshot (wifi signal in the top left is actually connected to a mobile 3G hub) from an iPhone SE, so it works :smile 

![14996432_622387437941344_54008383_n](https://cloud.githubusercontent.com/assets/7526156/20025916/98262320-a2eb-11e6-894d-1e1b6791ac15.jpg)

#### You don't have to use port 8887 (it's recommended that you use high numbers) just as long as you match the "translate to" section to the same as your port you set on "200 OK"

(You can mark this as closed, I just wanted to share this so people googling might navigate to this issue and be helped).

**Comment by kzahel** [comment]
Cool! I'll link to it from the main project page later. Thanks for the writeup.

**Comment by kzahel** [comment]
I don't have a BT Home Hub, and not sure how many people do. But frequently you can get to your router settings page by navigating to http://192.168.0.1 or http://192.168.1.1 where you can configure port forwarding settings.

**Comment by Blue-EyesWhiteDragon** [comment]
@kzahel Yeah usually it's something like that; mine is configured by default to 192.168.1.254

**Comment by cbymc** [comment]
can I run a pocketmine server for mcpe on my chromebook with linux if i do this

**Comment by kzahel** [comment]
@cbymc you might be able to. There's a standalone app that could help you setup the port forwarding: https://chrome.google.com/webstore/detail/hetimaportmap/naifildeohmcocnmibaampijofhcohif

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
